### PR TITLE
curvefs/tool: delete-fs

### LINKED ISSUE #843: curvefs/tool: delete-fs add noconfirm
**Is your feature request related to a problem? (你需要的功能是否与某个问题有关?)**

**Describe the solution you'd like (描述你期望的解决方法)**

add -noconfirm for delete-fs

add output to remind user the delete couldn't been restore.

change confirm from 'y' to 'Yes, delete!'

**Describe alternatives you've considered (描述你想到的折衷方案)**

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
### What problem does this PR solve
1. add -noconfirm for delete-fs
2. adjust delete-fs confirm input
3. Readme fix

Issue Number: close #843  

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by Cyber-SiKu** [comment]
recheck

**Comment by Cyber-SiKu** [comment]
recheck

File: curvefs/src/tools/README.md [hunk 1]
```diff
@@ -294,6 +294,90 @@ metaserverId:2, hostname:pubbeta2-curve20.dg.163.org, hostIp:192.168.1.1, port:3
...
 
+> You can use -jsonType and -jsonPath to output the topology to a json file.
+>
+> The parameter -jsonType supports build and tree.
+>
+> build: can be used to create the same topology as the current one;
+>
+> tree: build a json tree of the current topology.
+>
+> Usage:
```
Comment (SeanHai) [inline curvefs/src/tools/README.md:305]: The default jsonType is tree?

File: curvefs/src/tools/README.md [hunk 2]
```diff
@@ -294,6 +294,90 @@ metaserverId:2, hostname:pubbeta2-curve20.dg.163.org, hostIp:192.168.1.1, port:3
...
+>      "externalip": "10.182.2.46",
+>      "internalport": 8900,
+>      "externalport": 8900,
+>      "internalip": "10.182.2.46",
+>      "name": "10.182.2.46_0_0",
+>      "pool": "defaultPool",
+>      "zone": "zone1"
+>    },
+>    {
+>      "externalip": "10.182.2.47",
```
Comment (SeanHai) [inline curvefs/src/tools/README.md:332]: Keep the ip and port same as above.

File: curvefs/src/tools/curvefs_tool_define.cpp [hunk 3]
```diff
@@ -42,7 +42,7 @@ DEFINE_string(poolId, "1,2,3", "pools id");
 DEFINE_bool(detail, false, "show more infomation");
 DEFINE_string(partitionId, "1,2,3", "partition id");
 DEFINE_string(metaserverId, "1,2,3", "metaserver id");
-DEFINE_bool(confirm, false, "confirm execution");
+DEFINE_bool(noconfirm, false, "executed command without confirmation");
```
Comment (SeanHai) [inline curvefs/src/tools/curvefs_tool_define.cpp:45]: executed -> execute

File: curvefs/src/tools/README.md [hunk 4]
```diff
@@ -294,6 +294,90 @@ metaserverId:2, hostname:pubbeta2-curve20.dg.163.org, hostIp:192.168.1.1, port:3
...
+>      "name": "10.182.2.47_1_0",
+>      "pool": "defaultPool",
+>      "zone": "zone2"
+>    },
+>    {
+>      "externalip": "10.182.2.48",
+>      "externalport": 8902,
+>      "internalip": "10.182.2.48",
+>      "internalport": 8902,
+>      "name": "10.182.2.48_2_0",
```
Comment (wu-hanqing) [inline curvefs/src/tools/README.md:345]: why not display hostname?

File: curvefs/src/tools/README.md (hunk 1 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:305]: > The default jsonType is tree?

build

File: curvefs/src/tools/README.md (hunk 2 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:332]: > Keep the ip and port same as above.

fix

File: curvefs/src/tools/curvefs_tool_define.cpp (hunk 3 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/curvefs_tool_define.cpp:45]: > 

fix

File: curvefs/src/tools/README.md (hunk 4 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:345]: > why not display hostname?

in server that is it's name.

**Review by SeanHai — APPROVED** [review]

**Comment by Cyber-SiKu** [comment]
recheck

**Review by wu-hanqing — APPROVED** [review]

**Pull request merged**